projects/netgrif-components-core/src/lib/data-fields/file-field/abstract-file-field.component.ts

```diff
@@ -197,8 +197,11 @@ export abstract class AbstractFileFieldComponent extends AbstractDataFieldCompon
         this.state.uploading = true;
         const fileFormData = new FormData();
         const fileToUpload = this.fileUploadEl.nativeElement.files.item(0) as File;
+        const data = {};
```

Comment:
Create an interface to define the structure for sent data. It should the same as the request body object on the backend side.

Reply:
done